src/main/java/uk/nhs/prm/e2etests/service/HealthCheckService.java

```diff
@@ -0,0 +1,61 @@
+package uk.nhs.prm.e2etests.service;
+
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
```

Comment:
Can we opt for single line imports over wildcard please!

Reply:
Agree! PR addressed :)